scilpy/tractanalysis/fibertube_scoring.py

```diff
@@ -381,9 +418,9 @@ def mean_reconstruction_error(centerlines, centerlines_length, diameters,
         process.
     streamlines_length: ndarray,
         Fixed array containing the number of coordinates of each streamline
-    seeds_fiber: list
+    fibertube_of_seeds: list
```

Comment:
Odd variable name, can't you think of something clearer? `seeded_fibertube_index` maybe?